web/pdf_viewer.js

```diff
@@ -856,12 +860,15 @@ var PDFViewer = (function pdfViewer() {
     /**
      * @param {HTMLDivElement} pageDiv
      * @param {PDFPage} pdfPage
+     * @param {boolean} renderInteractiveForms
      * @returns {AnnotationLayerBuilder}
      */
-    createAnnotationLayerBuilder: function (pageDiv, pdfPage) {
+    createAnnotationLayerBuilder: function (pageDiv, pdfPage,
+                                            renderInteractiveForms) {
       return new AnnotationLayerBuilder({
         pageDiv: pageDiv,
         pdfPage: pdfPage,
+        renderInteractiveForms: this.renderInteractiveForms,
```

Comment:
Since you're passing in `renderInteractiveForms` as a parameter, shouldn't this line be `renderInteractiveForms: renderInteractiveForms,` instead?
Otherwise there's really no point in providing that parameter to this method (compare also with the `createTextLayerBuilder` method just above).